### DISCUSSION TITLE
Installed it on windows 10 mistakenly, Now I can't see my taskbar & it isn't running & My background is completely blacked out!

### DISCUSSION THREAD

**Discussion by Droyk** [valinet/ExplorerPatcher#754] [Q&A] [answered]
task manager, run, cmd, & windows explorer working fine can't open settings can't use search and no keyboard shortcut that has a start (Windows logo key) as a modifier works even start + r! 
Any solutions would be greatly appreciated!

Edit: Fixed because @valinet thought about it! 
[>As it stands now, PLEASE do not try running this on Windows 10. There are no checks for preventing you from doing so and you will probably brick your system; not really, but you should be prepared with the knowledge to restore explorer to a non-crashing state **(i.e. delete dxgi.dll from C:\Windows BY RENAMING IT TO dxgio.dll FIRST, starting explorer, confirming it no longer crashing, and then deleting dxgio.dll**)](https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems#other-aspects)

**Comment by Droyk** [comment, marked as answer]
>ExplorerPatcher on Windows 10 is not supported. This should be pretty obvious from the headline of the project: "restore a productive working environment on Windows 11". I moved on from Windows 10 to Windows 11. I do not have time to fully test against both versions. I daily drive stable Windows 11 (builds based on 22000), so I test on that. If someone wants to help maintain a Windows 10 port, sure, they have all my support. As it stands now, PLEASE do not try running this on Windows 10. There are no checks for preventing you from doing so an you will probably brick your system; not really, but you should be prepared with knowledge to restore explorer to a non-crashing state (i.e. **delete dxgi.dll from C:\Windows BY RENAMING IT TO dxgio.dll FIRST, starting explorer, confirming it no longer crashing, and then deleting dxgio.dll**)

Thank you @valinet for thinking about it. I seriously had a heart attack there didn't have any recent restore points were seriously considering a reinstall.

**Comment by Droyk** [comment]
@valinet Please update the wiki reporting problem section by adding this location too `C:\WINDOWS\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\dxgi.dll`
to remove `dxgi.dll` otherwise start menu wouldn't start!

Here's what my event viewer error says 
```
Faulting application name: StartMenuExperienceHost.exe, version: 0.0.0.0, time stamp: 0xe40668d4
Faulting module name: dxgi.dll, version: 22000.469.42.2, time stamp: 0x61f35aa8
Exception code: 0xc0000005
Fault offset: 0x0000000000029be5
Faulting process id: 0x1afc
Faulting application start time: 0x01d8157179295227
Faulting application path: C:\WINDOWS\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\StartMenuExperienceHost.exe
Faulting module path: C:\WINDOWS\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\dxgi.dll
Report Id: e7323e7b-2967-40aa-b7cc-3739e5f2a8f5
Faulting package full name: Microsoft.Windows.StartMenuExperienceHost_10.0.19041.1023_neutral_neutral_cw5n1h2txyewy
Faulting package-relative application ID: App
```

**Reply by valinet** [reply]
It’s installs multiple things, not only that. After one gets Explorer back up and running, he/she can go and uninstall the program which will revert all the modifications. I added a mention for that.
